Call to `f32::exp()` prevents crate to compile on `no_std`

The `f32::exp()` method is part of the f32 primitive in `std` and unfortunately not part of `core`. It breaks compilation on my target (Arm Cortex, nRF52833). Removing the call to `.exp()` fixes compilation, so it seems this is the only place where a `std` function is called. 

```
   Compiling sgpc3 v0.0.2
error[E0599]: no method named `exp` found for type `f32` in the current scope
   --> /home/eflukx/.cargo/registry/src/github.com-1ecc6299db9ec823/sgpc3-0.0.2/src/lib.rs:415:69
    |
415 |             * (rh * 61.12_f32 * ((17.62_f32 * t / (243.12_f32 + t)).exp()) / (273.15_f32 + t));
    |                                                                     ^^^ method not found in `f32`
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -15,6 +15,7 @@ edition = "2018"
 [dependencies]
 sensirion-i2c = "0.1"
 embedded-hal = "0.2"
+num-traits = { version = "0.2.14", default-features = false, features = ["libm"] }
 
 [dev-dependencies]
 linux-embedded-hal = "0.3"

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -123,6 +123,8 @@
 //! SGPC3 is a great sensor and fun to use! I hope your sensor selection and this driver servers you well.
 #![cfg_attr(not(test), no_std)]
 
+use num_traits::real::Real;
+
 use embedded_hal as hal;
 
 use hal::blocking::delay::DelayMs;

